plugins/obs-transitions/transition-luma-wipe.c

```diff
@@ -98,9 +107,11 @@ static void *luma_wipe_create(obs_data_t *settings, obs_source_t *source)
 	lwipe->ep_a_tex    = gs_effect_get_param_by_name(effect, "a_tex");
 	lwipe->ep_b_tex    = gs_effect_get_param_by_name(effect, "b_tex");
 	lwipe->ep_l_tex    = gs_effect_get_param_by_name(effect, "l_tex");
+	lwipe->ep_l_color = gs_effect_get_param_by_name(effect, "l_color");
```

Comment:
Style:  Align the assignment operator (`=`) with the lines above and below it.

```suggestion
	lwipe->ep_l_color  = gs_effect_get_param_by_name(effect, "l_color");
```